API/Controllers/ProjectController.cs

```diff
@@ -358,21 +358,57 @@ public async Task<IActionResult> CreateProjectAsync([FromBody] ProjectResource p
                 return BadRequest(problem);
             }
 
-            if(projectResource.CallToAction != null)
+            if(projectResource.CallToActions != null)
             {
-                IEnumerable<CallToActionOption> callToActionOptions =
-                    await callToActionOptionService.GetCallToActionOptionFromValueAsync(
-                        projectResource.CallToAction.OptionValue);
-                if(!callToActionOptions.Any())
+                if(projectResource.CallToActions.GroupBy(cta => cta.OptionValue).Any(cta => cta.Count() > 1))
                 {
                     ProblemDetails problem = new ProblemDetails
                     {
-                        Title = "Call to action value was not found.",
-                        Detail = "The specified call to action value was not found while creating the project.",
-                        Instance = "40EE82EB-930F-40C8-AE94-0041F7573FE9"
+                        Title = "Duplicate call to action option value.",
+                        Detail = "It is not possible to create a project with multiple of the same call to actions.",
+                        Instance = "D2C8416A-9C55-408B-9468-F0E5C635F9B7"
                     };
                     return BadRequest(problem);
                 }
+
+                if(projectResource.CallToActions.GroupBy(cta => cta.Value).Any(cta => cta.Count() > 1))
+                {
+                    ProblemDetails problem = new ProblemDetails
+                    {
+                        Title = "Duplicate call to action value.",
+                        Detail = "It is not possible to create a project with multiple call to actions with the same values.",
+                        Instance = "965D9C50-064F-4AFD-8CEB-28E556E47906"
+                    };
+                    return BadRequest(problem);
+                }
```

Comment:
Shouldn't this check if there are duplicate `OptionValues` instead of `Values`? Option value is the type of call to action (e.g "Join today") and value is the url ("https://github.com/digitalexellence/dex-frontend"). Right now it's not possible to link the same page twice, I don't think that that is the behaviour we want.

Reply:
Well, as this is the second pull-request for this issue, it may be a little unclear. But in the previous PR #440 it was asked to also check for the values. So currently it's not possible to add any duplicate buttons as that could be unclear and it's not possible to add multiple of the same URL's.

Reply:
I think people should be able to add the same url on different buttons, maybe @RubenFricke & @niraymak can give their opinion on this?

Reply:
In my opinion, the URL can be the same in multiple added call to actions. However, you should not be able to add 2 the same call to action (I think I also mentioned this in the other pull request, but I can't remember anything said about checking on the values). @niraymak do you have any opinions about this?

Reply:
I agree with you. @RubenFricke

Reply:
Good, I'll update this today.